corehq/apps/users/models.py

```diff
@@ -1651,9 +1651,11 @@ def save(self, fire_signals=True, **params):
         super(CommCareUser, self).save(fire_signals=fire_signals, **params)
 
         if fire_signals:
+            from corehq.apps.callcenter.tasks import sync_user_cases
             from .signals import commcare_user_post_save
             results = commcare_user_post_save.send_robust(sender='couch_user', couch_user=self,
                                                           is_new_user=is_new_user)
+            sync_user_cases.delay(self)
```

Comment:
(fwiw, passing the ID is also good practice in case the instance can't be serialised when it's put on the queue. For us they're usually JsonObjects so not an issue, but potentially a gotcha.)